### PR TITLE
Fix CQ tests

### PR DESCRIPTION
**Pull request by huchenlei**
## Description

Make test client script also run on CPU.

## Checklist:

- [ ] I have read [contributing wiki page](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing)
- [ ] I have performed a self-review of my own code
- [ ] My code follows the [style guidelines](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing#code-style)
- [ ] My code passes [tests](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Tests)

### CODE REVIEW

**Comment by lllyasviel** [comment]
lots of preprocessors are broken now. what happened

**Comment by huchenlei** [comment]
Suspected PR reverted. I should probably not attempt to fix the linter in a batch without having test coverage up. There was likely some unused imports I removed that actually is doing something.

**Comment by huchenlei** [comment]
There seem to be issue loading empty model with the CQ setup.

```
 File "/home/runner/work/stable-diffusion-webui-forge/stable-diffusion-webui-forge/modules/sd_models.py", line 504, in get_sd_model
    load_model()
  File "/home/runner/work/stable-diffusion-webui-forge/stable-diffusion-webui-forge/modules/sd_models.py", line 580, in load_model
    sd_model = forge_loader.load_model_for_a1111(timer=timer, checkpoint_info=checkpoint_info, state_dict=state_dict)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/runner/work/stable-diffusion-webui-forge/stable-diffusion-webui-forge/modules_forge/forge_loader.py", line 151, in load_model_for_a1111
    forge_objects = load_checkpoint_guess_config(
  File "/home/runner/work/stable-diffusion-webui-forge/stable-diffusion-webui-forge/modules_forge/forge_loader.py", line 89, in load_checkpoint_guess_config
    model_config = model_detection.model_config_from_unet(sd, "model.diffusion_model.", unet_dtype)
  File "/home/runner/work/stable-diffusion-webui-forge/stable-diffusion-webui-forge/ldm_patched/modules/model_detection.py", line 163, in model_config_from_unet
    unet_config = detect_unet_config(state_dict, unet_key_prefix, dtype)
  File "/home/runner/work/stable-diffusion-webui-forge/stable-diffusion-webui-forge/ldm_patched/modules/model_detection.py", line 49, in detect_unet_config
    model_channels = state_dict['{}input_blocks.0.0.weight'.format(key_prefix)].shape[0]
KeyError: 'model.diffusion_model.input_blocks.0.0.weight'
```

File: test/test_face_restorers.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 @pytest.mark.usefixtures("initialize")
 @pytest.mark.parametrize("restorer_name", ["gfpgan", "codeformer"])
+@pytest.mark.skip  # Skip for forge.
```
Comment (huchenlei) [inline test/test_face_restorers.py:11]: Skip as need to download several large face detection models.

File: test/test_txt2img.py [hunk 2]
```diff
@@ -69,6 +69,7 @@ def test_txt2img_with_tiling_performed(url_txt2img, simple_txt2img_request):
     assert requests.post(url_txt2img, json=simple_txt2img_request).status_code == 200
 
 
+@pytest.mark.skip  # Skip for forge.
```
Comment (huchenlei) [inline test/test_txt2img.py:72]: Skip as need to download several large face detection models.

**Pull request merged**